sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -683,11 +684,36 @@ public HttpPipeline getHttpPipeline() {
         return protocolLayer.getDigitalTwinModels().deleteWithResponseAsync(modelId, context);
     }
 
-    //TODO: Decommission Model APIs (waiting for Abhipsa's change to come in)
     PagedFlux<String> listRelationships(String digitalTwinId, String relationshipName, Context context) {
         return new PagedFlux<>(
             () -> listRelationshipsFirstPage(digitalTwinId, relationshipName, context),
             nextLink -> listRelationshipsNextPage(nextLink, context));
     }
 
+    /**
+     * Decommissions a model.
+     * @param modelId The Id of the model to decommission.
+     * @return an empty Mono
```

Comment:
why we need to write this return?

Reply:
java doc analyzer will complain if there is no return section in the comments for publicly facing APIs